src/app.types.tsx

```diff
@@ -94,8 +94,33 @@ export interface AddCatalogueCategoryProperty {
   unit?: string;
   mandatory: boolean;
   allowed_values?: AllowedValues;
+  default_value?: string | number | boolean;
 }
 
+export interface CatalogueCategoryPropertyMigration {
```

Comment:
Similar for this, I assume its used for the forms only?

Reply:
yes